Asterisk.2013/Asterisk.NET/Manager/Action/ParkAction.cs

```diff
@@ -50,7 +54,12 @@ public override string Action
         /// <summary>
         ///     Set the Channel where the Call will end up after the timeout is reached.
         /// </summary>
-        public string Channel2 { get; set; }
+        public string TimeoutChannel { get; set; }
+
+        /// <summary>
+        ///     Set the Announcement Channel to receive the AMI Announcement.

```

Comment:
Maybe add the full parameter description shown on the Asterisk Documentation:
https://wiki.asterisk.org/wiki/display/AST/Asterisk+12+ManagerAction_Park
> Channel name to use when constructing the dial string that will be dialed if the parked channel times out. If TimeoutChannel is in a two party bridge with Channel, then TimeoutChannel will receive an announcement and be treated as having parked Channel in the same manner as the Park Call DTMF feature.